app/src/lib/getBearerToken.ts

```diff
@@ -0,0 +1,9 @@
+import { getCurrentUser } from "../localStorageManager";
+
+export const getBearerToken = () => {
```

Comment:
This is pulled out separately to make it easier to mock?

Reply:
yup and also it just gets a bearer token